docs/pages/kubernetes-access/helm/reference/teleport-kube-agent.mdx

```diff
@@ -388,6 +388,41 @@ This can be used for joining a Teleport agent to a Teleport cluster which does n
   One option might be to use Teleport's built-in [ACME support](./teleport-cluster.mdx#acme) or enable [cert-manager support](./teleport-cluster.mdx#highavailabilitycertmanager).
 </Admonition>
 
+## `tls`
+
+### `existingCASecretName`
+
+| Type | Default value |
+| - | - |
+| `string` | `""` |
+
+`tls.existingCASecretName` sets the `SSL_CERT_FILE` environment variable to load a trusted CA or bundle in PEM format into Teleport pods.
+This can be set to inject a root and/or intermediate CA so that Teleport can build a full trust chain on startup.
+
+You should create the secret in the same namespace as Teleport using a command like this:
+
+```shell
+kubectl create secret generic my-root-ca --from-file=ca.pem=/path/to/root-ca.pem
+```
+
+<Notice type="warning" title="Root CA filename">
+  The filename used for the root CA in the secret must be `ca.pem`.
```

Comment:
Do we need to edit the example command above so that the filename is `ca.pem`?

Reply:
The example above:
- creates a secret `my-root-ca`
- read the content of `/path/to/root-ca.pem`
- puts the content into a key called `ca.pem`
- when mounted by Kubernetes each key will become a file in the teleport pod
- the file created is called `ca.pem`

I will rephrase to avoid confusion